app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -40,13 +40,13 @@ register_host() {
        --data "host[name]=$(hostname --fqdn)" \
        --data "host[build]=false" \
        --data "host[managed]=false" \
-       <%= "--data \"host[organization_id]=#{@organization.id}\"" if @organization -%>
-       <%= "--data \"host[location_id]=#{@location.id}\"" if @location -%>
-       <%= "--data \"host[hostgroup_id]=#{@hostgroup.id}\"" if @hostgroup -%>
-       <%= "--data \"host[operatingsystem_id]=#{@operatingsystem.id}\"" if @operatingsystem -%>
-       <%= "--data \"setup_insights=#{@setup_insights}\"" unless @setup_insights.nil? -%>
-       <%= "--data \"setup_remote_execution=#{@setup_remote_execution}\"" unless @setup_remote_execution.nil? -%>
-       <%= "--data \"remote_execution_interface=#{@remote_execution_interface}\"" if @remote_execution_interface.present? -%>
+       <%= "--data 'host[organization_id]=#{@organization.id}' \\" if @organization -%>
+       <%= "--data 'host[location_id]=#{@location.id}' \\" if @location -%>
+       <%= "--data 'host[hostgroup_id]=#{@hostgroup.id}' \\" if @hostgroup -%>
+       <%= "--data 'host[operatingsystem_id]=#{@operatingsystem.id}' \\" if @operatingsystem -%>
+       <%= "--data 'setup_insights=#{@setup_insights}' \\" unless @setup_insights.nil? -%>
+       <%= "--data 'setup_remote_execution=#{@setup_remote_execution}' \\" unless @setup_remote_execution.nil? -%>
+       <%= "--data 'remote_execution_interface=#{@remote_execution_interface}' \\" if @remote_execution_interface.present? -%>
```

Comment:
I would move these to above the static ones. That means you don't need to think about which one is the last line. Otherwise you end up with a trailing `\` (so right after `<%= headers.join(' ') %>` and right before `--data "host[name]=$(hostname --fqdn)" \`). The same below. That allows you to remove the odd empty line as well.

Reply:
At the end I had to take another approach.

One problem is the prefix before `<%=`, if we want to properly align the output, we need to format it inside of the tag. Which is why I added white spaces inside of the string. I'm aware of `indent` macro, but the resulting line was harder to read

```
<%= "          --data 'host[organization_id]=#{@organization.id}' \\\n" if @organization -%>
```
vs
```
<%= indent(7) { "          --data 'host[organization_id]=#{@organization.id}' \\\n" } if @organization -%>
```

Second problem is, we can't use `%>` since it would always generate a new line even if the if condition is falsy. Therefore we need to keep `-%>` and render the new line from erb. Therefore "\\\n" at the end of each line.

With this approach, it does not really matter if it's above static part. The last `\` just makes `\n` insignificant but that does not break closing bracket `}`. The output is now aligned and readable.